### PR TITLE
UI Updates (part 2)

### PR DESCRIPTION
**Pull request by tarzzz**

### CODE REVIEW

**Comment by tarzzz** [comment]
Adds the following updates on top of @n-riesco's PR: https://github.com/plotly/falcon-sql-client/pull/198/

 On-Prem:
  - "/" now shows the Login Page (instead of link to login).
  - Removed Plot.ly and FAQ Tabs.

Webapp/Server version:
  - "/" now shows the Login Page (instead of link to login).
  - Removed FAQ

**Comment by tarzzz** [comment]
@chriddyp Please review.. This and #198 should resolves all the major concerns from the QA.. After these are merged, I am going to create updated binaries for a final review before the 2.3 release..

File: app/components/Settings/Settings.react.js [hunk 1]
```diff
@@ -420,8 +424,13 @@ class Settings extends Component {
                                                     <strong><code>{connectorUrl}</code></strong>
                                                 </Link>
                                             </div>
+                                            <p>
+                                                {`Logged in as "${this.props.username}"`}
+                                                <br/>
+                                                <a onClick={this.props.logout}>Log Out</a>
```
Comment (chriddyp) [inline app/components/Settings/Settings.react.js]: let's move this destructuring to the start of render where the rest of the `props` are destructured:
```
const {
   ....
   logout,
   ...
   username
} = this.props
```

File: app/components/Settings/Settings.react.js [hunk 2]
```diff
@@ -435,6 +444,19 @@ class Settings extends Component {
...
+                                            </p>
+                                        </div>
+                                    ): (
+                                        <div>
+                                            <p>
+                                                <a onClick={() => window.location.assign('/login')}>Log into Plotly</a>
+                                                <br/>
+                                                Please, login in order to connect your queries to your Plotly account.
```
Comment (chriddyp) [inline app/components/Settings/Settings.react.js]: > Log in to Plotly
> Log in to Plotly in order to schedule queries and make queries directly from the <a href="https://{onpremisedomain}/create>Plotly Chart Studio</a>

**Comment by chriddyp** [comment]
Just a couple small comments, then 💃

**Pull request merged**